packages/payload/src/queues/operations/runJobs/runJob/index.ts

```diff
@@ -60,6 +60,7 @@ export const runJob = async ({
     const errorJSON = hasFinalError
       ? {
           name: err.name,
+          cancelled: 'cancelled' in err && err.cancelled,
```

Comment:
Is `err.cancelled` possibly `undefined`? This should be wrapped in Boolean() it seems so that we don't return update job with `error.cancelled: undefined`

Reply:
done!